R/read.R

```diff
@@ -1,4 +1,4 @@
-st_read_agnostic <- function(dir, start_year, sectors) {
+st_read_agnostic <- function(dir, start_year, sectors, risktype) {
```

Comment:
nit: You call this variable risk_type when calling validate_input_values, I would suggest to stick to the name

Reply:
makes sense done